electrum/gui/kivy/main_window.py

```diff
@@ -1202,6 +1202,11 @@ def on_fee(self, event, *arg):
         self.set_fee_status()
 
     def protected(self, msg, f, args):
+        tx_btc = args[0].amount_msat/1000
+        msg += "\n\n" + _("This will send " + str(self.format_amount(tx_btc).strip()) + ' ' + self.base_unit )
```

Comment:
`_()` is a function that is used to translate strings. It can only translate static strings.
So if we want translation to work, instead of
`_("This will send " + str(self.format_amount(tx_btc).strip()) + ' ' + self.base_unit )`
you could do
`_("This will send {}").format(str(self.format_amount(tx_btc).strip()))  + ' ' + self.base_unit`

I haven't checked but why do we need `str(self.format_amount(tx_btc).strip())`, instead of just `self.format_amount(tx_btc)`? If the latter works, just do that.

See another example of string formatting with translation here:
https://github.com/spesmilo/electrum/blob/5891e039b1db74c0eed14fbf9b9f4e61d9f3546c/electrum/gui/qt/main_window.py#L1557

Reply:
Re the line breaks, see pic how the screen looks currently:
<details>
 <summary>pic</summary>

![Capture](https://user-images.githubusercontent.com/29142493/126490839-0b9f4b39-f0d2-4026-a6ce-df8a6217517a.PNG)
</details>

It's just a nit, but I suggest one fewer `\n` after `Pay lightning invoice?`,
and one more before `Enter your PIN code to proceed`.
i.e. there should be two paragraphs: first two sentences in first, `Enter your PIN code to proceed` in second.

Reply:
Hi, I have made the changes, please have a look again. 
Re the line breaks, I have kept 2 between all these lines to give a cleaner look, have a look and tell me if it's fine!

Reply:
![image](https://user-images.githubusercontent.com/33263976/126550857-f3d34141-f696-4cc7-beea-1552499a8785.png)